### PR TITLE
PLANET-2368 - Implement Author profile

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by comzeradd** [comment]
PR updated with @kirdia's commit. It's ready for review.

File: templates/tease-author.twig [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<li id="result-row-{{ post.ID }}" class="media search-result-list-item">
+	<img class="d-flex search-result-item-image" src="{{ post.thumbnail.src('thumbnail') }}"
+		 alt="{{ fn('esc_attr', post.thumbnail.alt|default( post.title ) )|raw }}" />
+
+	<div id="tease-{{ post.ID }}" class="media-body search-result-item-body tease tease-{{ post.post_type }}">
+		<div class="search-result-tags">
+			<span class="search-result-item-title">{{ post.get_custom_type }}</span>
+
+			{% for page_type in post.get_custom_terms %}
+				<button class="search-result-item-head no-btn" data-term_id="{{ page_type.term_id }}">{{ page_type.name }}</button>
```
Comment (angtheod) [inline templates/tease-author.twig]: there is missing js functionality for this, see [here](https://github.com/greenpeace/planet4-master-theme/blob/release-candidate/assets/js/search.js#L51)

File: templates/tease-author.twig [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<li id="result-row-{{ post.ID }}" class="media search-result-list-item">
+	<img class="d-flex search-result-item-image" src="{{ post.thumbnail.src('thumbnail') }}"
```
Comment (angtheod) [inline templates/tease-author.twig]: Same here, missing js funtctionality, see [here](https://github.com/greenpeace/planet4-master-theme/blob/release-candidate/assets/js/search.js#L58) and [here](https://github.com/greenpeace/planet4-master-theme/blob/release-candidate/assets/js/search.js#L63)

**Comment by comzeradd** [comment]
@angtheod I added a commit to fix these two issues. Given that we don't have a form to submit here, like in Search, I added the filter url in a more static way instead of using Javascript.

**Pull request merged**